### PR TITLE
Publish proper config.

### PR DESCRIPTION
**Pull request by lewisgoddard**

### CODE REVIEW

**Comment by emersion** [comment]
Not mergeable anymore.

**Comment by lewisgoddard** [comment]
@emersion Could you give this a look? Helpful for those of us that don't just use `router.php`.

File: nginx.conf [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 location / {
     rewrite "^/([a-z]{2}(?:_(?:[A-Z]{2}|[A-Z][a-z]+))?)/(.*)$" /$2?lang=$1 last;
     try_files $uri $uri.html $uri/ @extensionless-php;
 }
 
+location @extensionless-php {
+    rewrite ^(.*)$ $1.php last;
+}
+
+location ~ \.php$ {
```
Comment (emersion) [inline nginx.conf]: This is specific to the platform, is it needed? (e.g. not the same address on Debian and Arch)

**Comment by emersion** [comment]
I still get a 403 error when accessing to `/docs/`

**Comment by lewisgoddard** [comment]
I should be able to fix that by allowing index browsing.

File: nginx.conf (hunk 1 above)
Comment (lewisgoddard) [inline nginx.conf]: `location ~ \.php$ {` ?

I would have thought that applies to all Nginx installations.

Comment (emersion) [inline nginx.conf]: See https://wiki.archlinux.org/index.php/Nginx#nginx_configuration

**Pull request closed without merging**